int32 drone_id
int32 turns

Header header

int32 mat2d_num
Mat2d_33[9] mat2d_33

Mat2d_conf mat2d_conf

Mat3d mat3d